src/components/organisms/userFeedbackForm.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useEffect, useState } from "react";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import * as Sentry from "@sentry/react";
+import { AnimatePresence, motion } from "motion/react";
+import { useForm } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+
+import { LoggerService } from "@services/logger.service";
+import { namespaces } from "@src/constants";
+import { UserFeedbackFormProps } from "@src/interfaces/components";
+import { useToastStore } from "@src/store";
+import { cn } from "@src/utilities";
+import { userFeedbackSchema } from "@validations";
+
+import { Button, ErrorMessage, IconButton, Input, Loader, Textarea, Typography } from "@components/atoms";
+
+import { CatLookAtFishImage } from "@assets/image";
+import { Close } from "@assets/image/icons";
+
+export const UserFeedbackForm = ({ className, isOpen, onClose }: UserFeedbackFormProps) => {
+	const { t } = useTranslation("global", { keyPrefix: "userFeedback" });
+	const { t: tErrors } = useTranslation("errors");
+	const addToast = useToastStore((state) => state.addToast);
+	const [isSendingFeedback, setIsSendingFeedback] = useState(false);
+	const [isFeedbackSubmitted, setIsFeedbackSubmitted] = useState(false);
+	const [timerCount, setTimerCount] = useState(10);
+
+	const {
+		formState: { errors },
+		handleSubmit,
+		register,
+		reset,
+	} = useForm({
+		mode: "onChange",
+		defaultValues: {
+			name: "",
+			email: "",
+			message: "",
+		},
+		resolver: zodResolver(userFeedbackSchema),
+	});
+
+	const onSubmit = async (data: { email: string; message: string; name: string }) => {
+		if (isFeedbackSubmitted) {
+			onClose();
+
+			return;
+		}
+		try {
+			setIsSendingFeedback(true);
+			const { email, message, name } = data;
+			const sentryId = Sentry.captureMessage("Message that needs user feedback");
+			const userFeedback = { event_id: sentryId, name, email, message };
+
+			Sentry.captureFeedback(userFeedback);
+
+			setIsFeedbackSubmitted(true);
+			setTimerCount(10);
+		} catch (error) {
+			Sentry.captureException(error);
```

Comment:
`Sentry.captureException(error);` have a totally different purpose of error handling and not user feedback:
https://docs.sentry.io/platforms/javascript/usage/